Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -110,6 +110,52 @@ final class LayerTests: XCTestCase {
         XCTAssertEqual(output, expected)
     }
 
+    func testConv2DGradient() {
+        let filter =  Tensor(shape: [3, 3, 2, 4], scalars: (0..<72).map(Float.init))
+        let bias = Tensor<Float>(zeros: [4])
```

Comment:
Could you add a non-trivial bias?

Reply:
The values of the biases don't have an impact on the gradient. The gradient is only changed by the batch size of the inputs.